CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ Note that Sockeye has checks in place to not translate with an old model that wa
 
 Each version section may have have subsections for: _Added_, _Changed_, _Removed_, _Deprecated_, and _Fixed_.
 
+## [1.18.34]
+### Added
+- Constrains input from plain file. 
+- Contraints can be used for image captioning module. 
+- Zero padding of features enables to have input features of different shape for each image.
```

Comment:
Could you rephrase to make clear that this refers to image captioning? Something like: "Image Captioning: zero padding of features now allows input features of different shape for each image."

Reply:
Ok.